### PR TITLE
added font-size and font-weight to the drawer-item-open.disabled

### PR DESCRIPTION
**Pull request by ratnadeeprajput**

### CODE REVIEW

**Comment by moloko** [comment]
certainly fixes the issue for me, what I'm not qualified to judge is whether it's the best way to do it or not, I think it needs a CSS specialist to take a look

I can see that the reason the active PLP elements have a different style is because they are `<button>` elements whereas the inactive elements are `<div>` - which just makes me think that there might be some higher-level fix to be done here.

(I'm also confused by the presence of page-level-progress.less in the theme folder which suggest I'm not qualified to be reviewing this!!)

File: less/pageLevelProgress.less [hunk 1]
```diff
@@ -8,6 +8,8 @@
         &.drawer-item-open.disabled {
             background-color:@item-color-disabled;
             color:@item-text-color-disabled;
+            font-size: @button-text-font-size;
```
Comment (lc-thomasberger) [inline less/pageLevelProgress.less]: we can not use this variable in the plugin
it is defined only in the vanilla theme version 2, so build will break with older versions of the theme

**Comment by lc-thomasberger** [comment]
@moloko would be nice to use `<button disabled>` for the inactive PLP elements. However we can't properly style disabled elements in ie8.

For a CSS fix I would recommend applying the styles (font-size, font-weight) to page-level-progress.less in  adapt-contrib-vanilla.

**Comment by ratnadeeprajput** [comment]
PR raised - https://github.com/adaptlearning/adapt-contrib-vanilla/pull/186

**Comment by moloko** [comment]
is this still needed or should it be closed?

**Comment by chetan-hajare** [comment]
@ratnadeeprajput As the intended changes have been achieved in PR -https://github.com/adaptlearning/adapt-contrib-vanilla/pull/186 , we can close this PR.

**Pull request closed without merging**